shared/constants/transaction.ts

```diff
@@ -500,3 +506,19 @@ export enum TokenStandard {
   /** Not a token, but rather the base asset of the selected chain. */
   none = 'NONE',
 }
+
+/**
+ * Describes the event types emitted by the transaction controller.
+ */
+export enum TransactionEvent {
+  added = 'transaction-added',
+  approved = 'transaction-approved',
+  confirmed = 'transaction-confirmed',
+  dropped = 'transaction-dropped',
+  failed = 'transaction-failed',
+  newSwap = 'transaction-new-swap',
+  newSwapApproval = 'transaction-new-swap-approval',
+  rejected = 'transaction-rejected',
+  submitted = 'transaction-submitted',
+  postTransactionBalanceUpdated = 'post-transaction-balance-updated',
```

Comment:
Minor, but alphabetical?

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/21559/commits/7ef24787fd00bfd373c8209c9e6ea77bb2870a0f